internal/portforward/service/script.go

```diff
@@ -0,0 +1,29 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func (s *Service) runPortForwardedScript(ports []uint16) (err error) {
+	// run bash script with ports as arguments
+	portStrings := make([]string, len(ports))
+	for i, port := range ports {
+		portStrings[i] = fmt.Sprint(int(port))
+	}
+	portsString := strings.Join(portStrings, " ")
```

Comment:
we should have this as a comma separated value ideally, it's easier to split by comma then taking a variable number of arguments I think (across shells)